source/_posts/2019-02-20-why-we-invest-in-having-our-teams-visit-clients.markdown

```diff
@@ -0,0 +1,49 @@
+
+
+### It takes the relationship to a whole new level
+
+Let's be honest here, nothing beats being face to face to really bond with another person.
+
+When face to face, you actually chat about things regarding life itself that require a lot more effort to do remotely. In our history of visiting clients we had the pleasure to go out on different bonding activities like escape rooms, hiking, kayaking, you name it. Once my whole team went out to do archery together with the client, and after that we went to his house to eat a home cooked dinner his wife had prepared for us.
+
+By being able to engage in those things we really get to know the person we work with everyday. Nothing is the same after that. Once you can put a face with the name together, share experiences, and become aware of what each other likes and dislikes, a bond is created that sets the team up for success.
+
+### We end up adding much more value
+
+Even though we get hired because of our technical expertise, often we realize the bottlenecks preventing substantial progress may be how communication flows (or doesn't) within the team, or how smooth the dev process is (or isn't). Understanding where the real pain points reside by being observant let us help our clients in other areas of the business that need to be improved the most.
+  
+I don't need to go further back to come up with examples for this. Only a few weeks ago one of our teams got back from DC from a client visit. In just a few days we made important discussions happen, involving technical decisions as well as aligning the team with the company values; helped coming up with a plan to achieve the goals for the year according to their priorities, put some process in place to enable continuous improvement, and more importantly we did it in a way that allowed every voice to be heard. We not only provided everyone with tools to do their job better, but also helped them take a big step in the organizational change they're going through. [Here are a few pics of that trip](https://www.facebook.com/wyeworks/posts/10156516977498451) if you fancy.
+
+## Final thoughts
+I could keep on rambling about the benefits of having our teams work side by side with our clients, but it doesn't really make justice to how much everything changes after these visits. As I mentioned, we obviously get a lot from them, and so do our clients. It truly is a win-win situation.
+
+When you get a team to really bond together and share a common purpose, and most importantly understand that their strength comes from the team itself, then the result is unprecedented success.
```

Comment:
a-No se si entiendo mucho a que apunta: `most importantly understand that their strength comes from the team itself`.
b-`the result is unprecedented success` - Tendrá sentido reformularlo de forma tal de que diga algo onda: "está todo dado para el success" o algo asi. Digo esto pq hay otras cuestiones que hacen al success de un equipo, y que podrían no estar presentes en otros casos de equipos de gente que viaja y hace bonding.